src/net.cpp

```diff
@@ -962,6 +962,16 @@ static bool CompareNodeTXTime(const NodeEvictionCandidate &a, const NodeEviction
     return a.nTimeConnected > b.nTimeConnected;
 }
 
+
+//! Sort an array by the specified comparator, then remove the last K elements.
+template<typename T>
+static void removeLastKElements(std::vector<T> &elements, bool(*comparator)(const T&, const T&), size_t k)
+{
+    std::sort(elements.begin(), elements.end(), comparator);
+    size_t cutSize = std::min(k, elements.size());
```

Comment:
Naming convention in new code would call this `cut_size`. I would probably just inline this variable, though. Also I think it's a little weird how the code is using "remove" "cut" and "erase" interchangeably. I don't see why you wouldn't want to be consistent and just say "erase" everywhere.

Reply:
I like `k` to refer to the K in the function name. I agree that the naming should be more consistent, switching to 'erase' across the board.